clients/js/examples/node/app.js

```diff
@@ -0,0 +1,44 @@
+var fs = require('fs');
+var path = require('path');
+
+// var pathToClient = path.join(__dirname, '..', '..', 'dist', 'main', 'index');
+
+var express = require('express');
+// var chroma = require(pathToClient);
+var chroma = require('chromadb');
+var openai = require('openai');
+
+var app = express();
+app.get('/', async (req, res) => {
+  const cc = new chroma.ChromaClient("http://localhost:8000");
+  await cc.reset()
+
+  // const openAIembedder = new chroma.OpenAIEmbeddingFunction("key")
```

Comment:
nit: maybe a const BOOL at top of example to choose?

Reply:
I'm not sure what you mean? This seems ok for an example app.